spec/system/user/user_settings_response_templates_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe "User uses response templates settings", type: :system do
+  let!(:user) { create(:user) }
+  let(:response_template) { create(:response_template, user: user) }
+
+  context "when user is signed in" do
+    before do
+      sign_in user
+      response_template
+    end
+
+    context "when user has a response template already" do
+      it "can go to the edit page of the response template" do
+        visit "/settings/response-templates"
+        click_link "EDIT"
+        expect(page).to have_current_path "/settings/response-templates/#{response_template.id}", ignore_query: true
+      end
+
+      it "can delete a response template properly", js: true do
+        visit "/settings/response-templates"
+        click_button "DELETE"
+        accept_alert
```

Comment:
After a lot of head scratching about Capybara, I used `accept_alert` here since it was the only way this test seemed to work. I tried to follow the pattern I used previously but that didn't work either...

https://github.com/thepracticaldev/dev.to/blob/e6fda5e444ee087145ea6822fd20b23293e719bd/spec/system/organization/user_updates_org_settings_spec.rb#L18-L26